File: task_2_native_binding/native-image-view/android/src/main/java/expo/modules/nativeimageview/NativeImageViewModule.kt
package your.package.name

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class NativeImageViewModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("NativeImageView")

    View(NativeImageView::class) {
      Prop("imageUrl") { view: NativeImageView, url: String ->
        view.setImageUrl(url)
      }
    }
  }
}

File: task_2_native_binding/native-image-view/ios/NativeImageViewModule.swift
import ExpoModulesCore

public class NativeImageViewModule: Module {
  public func definition() -> ModuleDefinition {
    Name("NativeImageView")

    View(NativeImageView.self) {
      Prop("imageUrl") { (view: NativeImageView, url: URL) in
        view.setImageUrl(url)
      }
    }
  }
}